.github/workflows/cd.yml

```diff
@@ -0,0 +1,30 @@
+name: vue-vocabulary-cd
+
+on:
+  release:
+    types:
+    - published
+
+env:
+  NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: true
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.16.1'
+
+    - name: Install dependencies
+      run: npm install --no-audit
+
+    - name: Publish package
+      run: |
+        cd $GITHUB_WORKSPACE/vocabulary/
```

Comment:
This step is redundant. The working directory is already set to the project root.

Reply:
@dhruvkb I have a doubt. I examined the failed workflow logs, but couldn't figure out how can we say that the working directory points to the project root. Please help me.

Reply:
@dhruvkb I have been looking for the above doubt, and now found that the working directory is the root GitHub directory by default. Hence, I have made the necessary changes and removed the redundant line in PR https://github.com/creativecommons/vocabulary/pull/338. Once a test passes with this PR merged, I will make the necessary changes in the CD workflow file of this repo as well.